lib/charms/prometheus_k8s/v0/prometheus_scrape.py

```diff
@@ -2365,7 +2365,7 @@ def _get_tool_path(self) -> Optional[Path]:
         res = "cos-tool-{}".format(arch)
         try:
             path = Path(res).resolve()
-            path.chmod(0o777)
+            path.chmod(0o775)
```

Comment:
Should we do this in charmcraft.yaml?
https://github.com/canonical/prometheus-k8s-operator/blob/62cd2d9b8c3a528910ce12c553209817e1b7a889/charmcraft.yaml#L35

Reply:
We can do it *also* in `charmcraft.yaml` (to address the CIS Hardening rule for this charm), but not exclusively; otherwise, the library will set the permission to `0o775` for every charm that uses it.